models/classes/export/AbstractTestExport.php

```diff
@@ -46,30 +45,23 @@
 abstract class AbstractTestExport implements ExportHandler, PhpSerializable
 {
     use PhpSerializeStateless;
-    use EventManagerAwareTrait;
+    use EventManagerAwareTrait; // todo: call firing event via method to have abilty to override
```

Comment:
To override for what purpose? Do you have a use case in mind?

Reply:
I've already done it, but I forgot to remove the comment.
https://github.com/oat-sa/extension-tao-testqti/pull/2329/files#diff-1fc574c1b6d9f7e863bd7ffd50bb016655b86e5f60328619245741da86ab2d5bR135